### PR TITLE
Updated esm support for nodejs

### PR DESCRIPTION
**Pull request by ezioda004**

### CODE REVIEW

**Comment by ezioda004** [comment]
@googlebot I signed it!

File: src/features/modules.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ JavaScript modules are now [supported in all major browsers](https://caniuse.com
 <feature-support chrome="61"
                  firefox="60"
                  safari="11"
-                 nodejs="no"
+                 nodejs="13"
```
Comment (chicoxyzzy) [inline src/features/modules.md]: I think this should be
```suggestion
                 nodejs="13.2"
```
or whatever version it will be released

Comment (mathiasbynens) [reply src/features/modules.md]: +1

Comment (ezioda004) [reply src/features/modules.md]: Done

**Pull request merged**